plugins/riemann-controller/riemann_controller/tasks.py

```diff
@@ -80,9 +81,46 @@ def _deployment_config_dir():
 def _publish_configuration_event(state, deployment_config_dir_path):
     manager_queue = 'manager-riemann'
     exchange_name = 'cloudify-monitoring'
+
+    cloudify_agent = ctx.bootstrap_context.cloudify_agent
+
+    # This is actually used in the management worker, so the worker_conf
+    # should exist
+    try:
+        # If we're in an environment with celery we should have a
+        # worker configuration, but we'll check for backwards
+        # compatibility
+        import worker_conf
+        broker_cert_path = worker_conf.broker_cert_path
+    except (ImportError, AttributeError):
+        broker_cert_path = ''
+
+    if broker_cert_path == '':
+        # No SSL for rabbit
+        broker_port = 5672
+        ssl_enabled = False
+        ssl_options = {}
+    else:
+        broker_port = 5671
+        ssl_enabled = True
```

Comment:
we're adding an input for rabbitmq_ssl_enabled (or whatever else name) so ssl_enabled should be deduced from there.